packages/datastore/src/datastore.ts

```diff
@@ -32,15 +48,68 @@ import {
  *
  * https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type
  * https://hal.inria.fr/file/index/docid/555588/filename/techreport.pdf
+ *
+ * The internal algorithms require transactions to be delivered in causal
+ * order to guarantee stability. E.g. a transaction that removes some text
+ * must be delivered *after* the transaction that adds that text. Any
+ * deviation from this will *not* raise an error, but can lead to
+ * diverging state between peers.
```

Comment:
@sccolbert Regarding this comment: As far as I read the textfield logic (and saw it in action), any attempt to remove text that has not yet been added will be discarded, and the text will remain in the value. Maybe that is because the `cemetery` attribute of the textfield metadata is not in use? I'll see if I can create an MWE on Monday, so we have something specific to discuss, but I thought I would leave this note for now.